nextflow_schema.json

```
@@ -168,24 +169,42 @@
             "properties": {
                 "assembler": {
                     "type": "string",
-                    "default": "megahit",
-                    "enum": ["megahit", "rnaspades"],
-                    "description": "Specify which assembler you would like to run, possible alternatives: megahit, rnaspades. default: megahit",
+                    "enum": ["megahit", "spades"],
+                    "description": "Specify which assembler you would like to run, possible alternatives: megahit, spades. default: megahit",
```

```suggestion
                    "description": "Specify which assembler you would like to run, possible alternatives: megahit, spades.",
```
Actually there is no default set in nextflow.config (and you removed the "default: megahit" from schema.